xds/src/test/java/io/grpc/xds/internal/sds/trust/SdsX509TrustManagerTest.java

```diff
@@ -83,6 +86,186 @@ public void emptySanListContextTest() throws CertificateException, IOException {
     trustManager.verifySubjectAltNameInChain(certs);
   }
 
+  @Test
+  public void missingPeerCerts() throws CertificateException, FileNotFoundException {
```

Comment:
The exception is already caught, shouldn't declare in `throws`

Reply:
fixed